recipes/macholib/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "macholib" %}
+{% set version = "1.8" %}
+{% set sha256 = "323c9c8b85768244554b3c040808ed6393c783aa6eb1122e04dc8905f442e559" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
Not sure that this makes sense given the library is designed to parse macOS libraries. According the [trove classifier]( https://bitbucket.org/ronaldoussoren/macholib/src/53d85321baffbfd7d911572b5d37fbaa53056e5a/setup.cfg?at=default&fileviewer=file-view-default#setup.cfg-18 ), it seems the author only intends for this to work on macOS.

Reply:
From the readme: "Though this tool targets a platform specific file format, it is pure python code that is platform and endian independent."

Reply:
Yes, that's why I thought it might be ok making this noarch. Another reason is that PyInstaller (that we were trying to package) [specifies macholib as dependency](https://github.com/pyinstaller/pyinstaller/blob/162fe92d5e852ba526332b916de621363a5f2995/setup.py#L25) for all OSs. So since it appears to be also installing it in tests, I wasn't sure that not providing it e.g. on Linux would work (it might just raise an import error)...